test/test_torch.py

```diff
@@ -13415,6 +13415,26 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    def test_index_add(self, device):
+        # test coverage for issue with atomic add:
+        # https://github.com/pytorch/pytorch/issues/29475
+        def is_signed(dtype):
+            return torch.is_signed(torch.tensor(0, dtype=dtype))
```

Comment:
nit: you can actually create a very empty tensor with `torch.empty([], dtype=dtype)`. It's marginally faster than creating a tensor with data because it doesn't have to allocate any data.